### PR TITLE
Aggregation generalize

### PR DESCRIPTION
**Pull request by jlark**
**Updates**
- Power outages were calculated incorrectly, this is now fixed with better algorithm
- Modified sesh_site to have options around solar grid and batteries. 
- Data aggregation takes site settings into account when aggregating.

**TODO**:
- Reports should take the new Sesh_Site settings into acccount when offering report values
ie: if the site has no grid, grid outage shouldn't be on the list of report values. @upatricck

### CODE REVIEW

**Comment by jlark** [comment]
Ready for review! @upatricck

File: seshdash/tasks.py [hunk 1]
```diff
@@ -376,21 +379,12 @@ def find_chunks(input_list,key):
...
-            if i == (len(input_list)-2):
-            # We are at end of list
-
-                result_list.append(section)
-        else:
-            count = 0
-            section = {}
-            result_list.append(section)
-
+    print "got list ##### %s"%input_list
```
Comment (upatricck) [inline seshdash/tasks.py]: Removing the print statement.

File: seshdash/tests/data_generation.py [hunk 2]
```diff
@@ -30,6 +30,14 @@ def get_random_binary():
         return 1
     return 0
 
+def grid_data():
+    """
+    harcoded grid data
+    """
+
+
```
Comment (upatricck) [inline seshdash/tests/data_generation.py]: Deleteting function

Comment (jlark) [inline seshdash/tests/data_generation.py]: done

File: seshdash/tasks.py (hunk 1 above)
Comment (jlark) [inline seshdash/tasks.py]: done

**Pull request merged**